Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.4
+
+# Run as non-root, eventually
+RUN groupadd -r slamon && useradd -r -d /workspace -m -g slamon slamon 
+
+# Install dependencies
+RUN pip install requests python-dateutil responses
+
+# Install SLAMon Python Agent
+ADD . /workspace
+WORKDIR /workspace
+RUN pip install .
+RUN chown -R slamon:slamon /workspace
```

Comment:
True with the sources, but I think home directory is useful if we want to save logs. They can be easily exported from the container if they are in a simple and explicit location like `/workspace/agent.log`